Source/WebCore/rendering/RenderFlexibleBox.cpp

```diff
@@ -2385,4 +2384,10 @@ void RenderFlexibleBox::layoutUsingFlexFormattingContext()
     updateLogicalHeight();
 }
 
+bool RenderFlexibleBox::childIsParticipatingInBaselineAlignment(const RenderBox& child) const
+{
+    auto alignment = alignmentForChild(child);
+    return (alignment == ItemPosition::Baseline || alignment == ItemPosition::LastBaseline) && !hasAutoMarginsInCrossAxis(child);
+}
```

Comment:
This method only has a single caller and the logic is short, so adding it seems unnecessary to me.
By removing it you can reuse the `alignmentForChild(child)`.

Reply:
Agreed. It was useful before, but now after the cleanup it is much less useful. Changed.

Reply:
Removed